iosApp/iosApp/ContentViewController.swift

```diff
@@ -173,7 +173,7 @@ class ContentViewController: UIViewController {
                 print("Socket <error>. code: <\(error.code.description)> , message: <\(error.message ?? "No message")>")
                 self?.info.text = "Socket <error>. code: <\(error.code.description)> , message: <\(error.message ?? "No message")>"
             default:
-                break
+                print("Unexpected stateListener state: \(state)")
```

Comment:
Side note. 
Unfortunately Kotlin\Native compiler is unable to convert sealed classes to be 'exhausted` with Swift switch statements like it does for Kotlin\Jvm.